backend/src/main/resources/graphql/admin.graphqls

```diff
@@ -87,4 +87,5 @@ extend type Mutation {
   setRegistrationLinkIsDeleted(link: String, deleted: Boolean!): String
   adminUpdateOrganization(name: String!, type: String!): String
   markFacilityAsDeleted(facilityId: ID!, deleted: Boolean!): String
+  markOrganizationAsDeleted(organizationId: ID!, deleted: Boolean!): String
```

Comment:
I wish the naming was better, it indicates you are setting the org as deleted, but in fact, you can use it to un-delete an org if you pass `false` for the `deleted` param. Nonetheless, I see you are following the pattern of `markFacilityAsDeleted` and `organizationService.markOrganizationAsDeleted` so I get where you're coming from